apps/org/views.py

```diff
@@ -653,3 +655,34 @@ def get_organization(self, request, org_slug):
     def post(self, request, *args, **kwargs):
         from django.http import HttpResponseNotAllowed
         return HttpResponseNotAllowed('GET')
+
+
+class LocalUserAPI(LoginRequiredMixin, View):
+    ''' Setting up a local endpoint for users here. '''
+    def get(self, request, *args, **kwargs):
+        # picking the first organization here
+        org_id = request.user.organization_set.all().first().id
+
+        members = Member.objects.filter(user__organization=org_id)
+        members_data = list(members.values())
+
+        user_id_list = [mem.user.id for mem in members]
+
+        users = get_user_model().objects.filter(pk__in=user_id_list)
+        user_data = {user['id']: user for user in list(users.values())}
+
+        # TODO: Save all the extra_data from vmi when we create member and link to VMI, including `picture`
+
+        social_auth_data = list(get_user_model().social_auth.rel.related_model.objects.filter(user__pk__in=user_id_list).values())
+        extra_data = {user['user_id']: user['extra_data'] for user in social_auth_data}
+
+        for member in members_data:
+            member['user'] = user_data.get(member['user_id'], {})
+            member['extra_data'] = extra_data.get(member['user_id'], {})
```

Comment:
this is fine but a nice thing to do in the future would be moving this serializing data into a Serializer (either making this a DRF function and using their serializing patterns, or creating something else that serves the same purpose.)

Reply:
+1 

Thinking of this as a lightweight temporary solution and adding DRF when the api becomes a little more involved